File: src/main/java/um/haberes/report/model/haberes/core/DependenciaDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DependenciaDto {

    private Integer dependenciaId;

    private String nombre = "";

    private String acronimo = "";

    private Integer facultadId;

    private Integer geograficaId;

    private FacultadDto facultad;

    private GeograficaDto geografica;

}

File: src/main/java/um/haberes/report/kotlin/dto/DependenciaDto.kt
package um.haberes.report.kotlin.dto

data class DependenciaDto(

    var dependenciaId: Int? = null,
    var nombre: String = "",
    var acronimo: String = "",
    var facultadId: Int? = null,
    var geograficaId: Int? = null,
    var facultad: FacultadDto? = null,
    var geografica: GeograficaDto? = null

)
